internal/neofs/neofs.go

```diff
@@ -468,6 +545,16 @@ func (x *NeoFS) DeleteObject(ctx context.Context, prm layer.PrmObjectDelete) err
 	return nil
 }
 
+// MaxObjectSize returns max single object size in bytes in NeoFS.
```

Comment:
not really: it returns configured payload size limit for object slicing when enabled. And i doubt this is needed at all

Reply:
Updated